hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerWithMultiWriter.java

```diff
@@ -72,8 +73,12 @@ public class TestHoodieDeltaStreamerWithMultiWriter extends SparkClientFunctiona
   String basePath;
   String propsFilePath;
   String tableBasePath;
+  
+  @AfterEach
+  public void teardown() throws Exception {
+    TestDataSource.resetDataGen();
```

Comment:
try apply this to `TestHoodieDeltaStreamer` too? and any other usage of this?

Reply:
TestHoodieDeltaStreamer extends HoodieDeltaStreamerTestBase which extends UtilitiesTestBase which has this method as well. All the other usages of TestDataSource inherit from HoodieDeltaStreamerTestBase. So I don't think it needs to be added anywhere else

Reply:
shoudn't it be done in `org.apache.hudi.utilities.deltastreamer.HoodieDeltaStreamerTestBase#resetTestDataSource()` ?